Add DuplicateTabFinder extension

PawiX25 opened the pull request:
### Project Details
1. **What is the name of your project?**  
   DuplicateTabFinder

2. **How many hours did you spend developing this?**  
   25 hours

3. **What problem does your extension solve, or what makes it unique?**  
   Duplicate Tab Finder efficiently identifies and manages duplicate browser tabs by automatically detecting duplicates across all open windows, grouping them by domain, and offering customizable settings such as auto-closing duplicates and theme toggles. This helps reduce browser clutter, enhances productivity, and optimizes system performance.

4. **How many people have tested your extension?**  
   5

5. **Does your extension work in both Chrome and Firefox?**  
   Yes, it is compatible with both Chrome and Firefox browsers.

---

### Additional Comments
No additional comments.

JosiasAurel reviewed (changes requested):
please fix these errors. it appears you're using a feature that's limited to manifest version 3

<img width="796" alt="image" src="https://github.com/user-attachments/assets/65e12748-c3bd-4cec-9115-18a622c38087">

PawiX25 commented:
Hi Josias, I have updated the manifest to support Chrome and added a Firefox-specific manifest for Firefox-specific features. Thanks for understanding!

JosiasAurel reviewed (approved):
Sorry for the delay. If you haven't yet, please fill [this form](https://airtable.com/appgtKZwZBoOeFU7O/pagyeS51zr2kC1rp0/form) to get your grant.
